### DISCUSSION TITLE
是否有考虑开发ios系统软件？

### DISCUSSION THREAD

**Discussion by yuyu1025** [EasyTier/EasyTier#448] [Ideas]
目前的问题是只能在电脑端使用，而zerotier可以在手机端使用，这在使用nas进行同步设备数据时非常有用

**Comment by KKRainbow** [comment]
没钱啊，Apple Dev 很贵的。

先用 WireGuard 接入凑合吧
https://www.easytier.top/guide/network/use-easytier-with-wireguard-client.html

**Reply by yuyu1025** [reply]
另外在群晖中使用的比较多，有没有群晖的方案？

**Reply by KKRainbow** [reply]
好像矿神有群晖套件

**Reply by ifplusor** [reply]
没记错的话，一年600多

**Reply by KKRainbow** [reply]
有点贵

**Reply by yuyu1025** [reply]
感觉可以开源+付费模式。类似于cryptomator这种模式，代码开源，如果有动手能力自签也可以免费使用。但是App Store直接下载使用要付费，这样满足大部分人，也能带来一点点收入，不至于纯粹为爱发电。

**Reply by rainpaper-bs** [reply]
> 感觉可以开源+付费模式。类似于cryptomator这种模式，代码开源，如果有动手能力自签也可以免费使用。但是App Store直接下载使用要付费，这样满足大部分人，也能带来一点点收入，不至于纯粹为爱发电。

那万一app store没人买不就寄了吗

**Reply by mzbgf** [reply]
> 没钱啊，Apple Dev 很贵的。
> 
> 先用 WireGuard 接入凑合吧 https://www.easytier.top/guide/network/use-easytier-with-wireguard-client.html

不用上架App Store的, 发布ipa让用户自签就行

**Reply by undefine-man** [reply]
我相信用户中会有有这个dev账号的开发者的

**Comment by r4nchy** [comment]
apple users are paying consumers. Make the underlying core free but add a subscription system for something like Notification features or something. Kind of what https://github.com/ruddarr/app does. They have appstore version that has core-feature free but notification service has a minimal subscription. But they also have opensourced the full code along with the APNS. So anyone can sign their own ipa.

This was the appstore version can pay for itself. I mean thats the theory, I don't know if it will work out

**Comment by happytrudy** [comment]
你得打钱

**Comment by KKRainbow** [comment]
https://github.com/EasyTier/EasyTier-iOS

**Reply by huangqingchao** [reply]
这执行力，简直拉满了，拉源码学习学习下 SwiftUI 去。
